src/main/kotlin/no/nav/familie/ef/mottak/config/FlywayConfig.kt

```diff
@@ -8,7 +8,7 @@ import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
-@ConditionalOnProperty("spring.flyway.enabled")
+@ConditionalOnProperty(name = ["spring.cloud.vault.enabled"])
```

Comment:
Hva gjør denne?

Reply:
Det vi egentlig gjør er å bytte databasebruker for flyway. Det gir ikke mening uten flyway, men det gir heller ikke mening med mindre du kjører mot en vault-kontrollert database.